samples/AKS Tutorial/deploy.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: akstest
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: akstest
+  template:
+    metadata:
+      labels:
+        app: akstest
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: publishedapp
+        image: mcr.microsoft.com/dotnet/samples:aspnetapp
+        imagePullPolicy: Always
+        env:
+        - name: ASPNETCORE_URLS
+          value: http://+:80
+        - name: DOTNET_DiagnosticPorts
+          value: /diag/port.sock
+        volumeMounts:
+        - mountPath: /diag
+          name: diagvol
+        resources:
+          limits:
+            cpu: 250m
+            memory: 512Mi
+      - name: monitor
+        image: mcr.microsoft.com/dotnet/monitor:6.2.2
```

Comment:
For managed identity, we need at least 6.2, but I wasn't sure if we should use a specific version here or just the latest. My only concern with latest is breaking changes in the future that could cause things not to work for people following along, but I also don't know if we want to lock this to one specific version of 6.2.*

Reply:
You can use `mcr.microsoft.com/dotnet/monitor:6` which points to the latest 6.*.* version.